### PR TITLE
Add chaos test coverage for ARM

### PR DESCRIPTION
**Pull request by chaitanyaenr**
This run chaos tests on ARM OCP 4.14

### CODE REVIEW

**Comment by chaitanyaenr** [comment]
/pj-reharse periodic-ci-redhat-chaos-krkn-hub-main-arm-krkn-hub-tests-arm

**Comment by chaitanyaenr** [comment]
/pj-reharse periodic-ci-redhat-chaos-krkn-hub-main-arm-krkn-hub-tests-arm

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-arm-krkn-hub-tests-arm

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-arm-krkn-hub-tests-arm

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-arm-krkn-hub-tests-arm

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-arm-krkn-hub-tests-arm

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-arm-krkn-hub-tests-arm

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-arm-krkn-hub-tests-arm

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-arm-krkn-hub-tests-arm

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-arm-krkn-hub-tests-arm

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-arm-krkn-hub-tests-arm

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-arm-krkn-hub-tests-arm

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-arm-krkn-hub-tests-arm

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-arm-krkn-hub-tests-arm

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-arm-krkn-hub-tests-arm

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-arm-krkn-hub-tests-arm

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-arm-krkn-hub-tests-arm

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-arm-krkn-hub-tests-arm

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-arm-krkn-hub-tests-arm

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-arm-krkn-hub-tests-arm

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-arm-krkn-hub-tests-arm

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-arm-krkn-hub-tests-arm

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-arm-krkn-hub-tests-arm

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-arm-krkn-hub-tests-arm

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-arm-krkn-hub-tests-arm

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-arm-krkn-hub-tests-arm

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-arm-krkn-hub-tests-arm

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-arm-krkn-hub-tests-arm

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-arm-krkn-hub-tests-arm

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-arm-krkn-hub-tests-arm

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-arm-krkn-hub-tests-arm

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-arm-krkn-hub-tests-arm

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-arm-krkn-hub-tests-arm

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-arm-krkn-hub-tests-arm

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-arm-krkn-hub-tests-arm

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-arm-krkn-hub-tests-arm

File: ci-operator/jobs/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main-periodics.yaml [hunk 1]
```diff
@@ -345,17 +374,6 @@ periodics:
     ci.openshift.io/generator: prowgen
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests-413
-  reporter_config:
```
Comment (paigerube14) [inline ci-operator/jobs/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main-periodics.yaml:348]: can we add this back in?

File: ci-operator/jobs/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main-periodics.yaml [hunk 2]
```diff
@@ -248,6 +215,79 @@ periodics:
...
+    repo: krkn-hub
+  labels:
+    ci-operator.openshift.io/cloud: aws
+    ci-operator.openshift.io/cloud-cluster-profile: aws-perfscale-qe
+    ci-operator.openshift.io/variant: arm
+    ci.openshift.io/generator: prowgen
+    job-release: "4.13"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-redhat-chaos-krkn-hub-main-arm-krkn-hub-tests-arm
+  spec:
```
Comment (paigerube14) [inline ci-operator/jobs/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main-periodics.yaml:234]: let's add the slack integration into the new tests we added

File: ci-operator/config/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main__4.14-nightly-x86-64.yaml [hunk 3]
```diff
@@ -40,7 +40,7 @@ tests:
     pre:
     - chain: ipi-aws-pre
     - chain: create-infra-move-ingress-monitoring-registry
-    workflow: redhat-chaos-krkn-hub-tests
+    workflow: redhat-chaos-pvc-scenarios
```
Comment (paigerube14) [inline ci-operator/config/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main__4.14-nightly-x86-64.yaml:43]: shouldn't we run both tests here? like you did below?

File: ci-operator/config/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main__4.14-nightly-x86-64.yaml [hunk 4]
```diff
@@ -54,7 +54,9 @@ tests:
     pre:
     - chain: ipi-aws-pre
     - chain: create-infra-move-ingress-monitoring-registry
-    workflow: redhat-chaos-hog-scenarios
+    test:
+    - chain: redhat-chaos-krkn-hub-tests
```
Comment (paigerube14) [inline ci-operator/config/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main__4.14-nightly-x86-64.yaml:58]: think we can run both these tests in above test run and keep this as the hog scenario run

File: ci-operator/config/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main__4.14-nightly.yaml [hunk 5]
```diff
@@ -38,7 +38,9 @@ tests:
     pre:
     - chain: ipi-aws-pre
     - chain: create-infra-move-ingress-monitoring-registry
-    workflow: redhat-chaos-krkn-hub-tests
+    test:
```
Comment (paigerube14) [inline ci-operator/config/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main__4.14-nightly.yaml:41]: depending on which gets in first we will have to edit the HCP/rosa files too 

https://github.com/openshift/release/pull/42613

File: ci-operator/config/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main__4.14-nightly-x86-64.yaml (hunk 4 above)
Comment (chaitanyaenr) [reply ci-operator/config/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main__4.14-nightly-x86-64.yaml]: Done!

File: ci-operator/config/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main__4.14-nightly-x86-64.yaml (hunk 3 above)
Comment (chaitanyaenr) [reply ci-operator/config/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main__4.14-nightly-x86-64.yaml]: Fixed it. Thanks.

File: ci-operator/jobs/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main-periodics.yaml (hunk 2 above)
Comment (chaitanyaenr) [reply ci-operator/jobs/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main-periodics.yaml:278]: Done!

File: ci-operator/jobs/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main-periodics.yaml (hunk 1 above)
Comment (chaitanyaenr) [reply ci-operator/jobs/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main-periodics.yaml]: Done!

**Pull request merged**